spec/validation_spec.rb

```diff
@@ -389,3 +389,52 @@
     expect(dupe_names).to be_empty
   end
 end
+
+RSpec.describe 'Version translation from v2 to v3' do
+  before :all do
+    # TODO: DRY this up --- duplicate code from example validators spec above
+  
+    # Nokogiri doesn't seem to support XSDs which import other schemas with URLs
+    # for the schemaLocation. To allow testing, we download the imported schema
+    # and point schemaLocation to it instead
+    schema_doc = Nokogiri::XML(File.read('BuildingSync.xsd'))
+
+    GEOJSON_XSD_PATH = 'geojson.xsd'
```

Comment:
Isn't this supposed to be GBXML_XSD_PATH and not GEOJSON?

Reply:
whoops, not sure how that happened, I'll get that fixed

Reply:
terminology leaking from other projects :(